lib/Installation/Partitioner/FormattingOptionsPage.pm

```diff
@@ -88,7 +88,7 @@ sub select_filesystem {
     wait_screen_change(sub {
             send_key 'end';
     }, 20);
-    send_key_until_needlematch((sprintf FILESYSTEM_TYPE, $filesystem), 'up');
+    send_key_until_needlematch((sprintf FILESYSTEM_TYPE, $filesystem), 'up', timeout => 15);
```

Comment:
2 things, first as per documentation http://open.qa/api/testapi/#_send_key_until_needlematch, `timeout` is positional argument.
Second thing, default timeout is 1 second and it works sometimes with that value. Bumping it to 15 might be too much.

Reply:
i used 5 but i hit the issue again. So i put another bigger number. i am thinking to try with 10 but it would be still considered too much IMO.

Reply:
Please, see the first point.